tests/flow/test_graph_deletion.py

```
@@ -557,6 +558,132 @@ def test25_delete_does_not_leave_phantom_label_entries(self):
+        # reserve_relationship() hands out an id from the graph's deleted set
+        # without clearing it there, so for the whole life of the pending create
+        # the id still reads as deleted. DELETE used to test only that flag and
+        # so skipped the edge, leaving the create standing: the edge survived the
+        # very query that deleted it. Ids are only recycled once something has
+        # been deleted, which is why the first iteration below used to pass and
+        # every later one leaked.
+        self.graph.delete()
+
+        self.graph.query("CREATE (:P {id: 1}), (:P {id: 2})")
+
+        for i in range(6):
```

Ruff (0.16.2)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)